src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -109,8 +112,11 @@ const noticeCss = {
     marginBottom: 3,
 };
 
+let dlTitleIdx = 0;
+
 export function DownloadOptionPanel (props) {
     const {groupKey, cutoutSize, help_id, children, style, title, tbl_id, dlParams, dataTag} = props;
+    const { showZipStructure=true, showEmailNotify=true, showFileLocation=true, showTitle=true } = props;
```

Comment:
Does showFileLocation=true not cause issue in LSSTCatalogSelectViewPanel?  LSSTCatalogSelectViewPanel is the only NONE IRSA application which uses DownloadOptionPanel.   LSSTCatalogSelectViewPanel does not have workspace.  I forgot how to build LSST so I am not able to test it.

Reply:
It should not be an issue since the panel will only appear when `useWs` is also true. 
But, this remind me I need to show Save As even if workspace is not used.  I will add this into the PR.

Reply:
Yes, the `label` is fine (even though CCB could argue to change it to something else in the future) but the value is the one you can pass outside and this one will contain time stamp. So i think we are good. @lznakano should confirm once you merge.